microservices/matchmaking-service/src/scrim/scrim.event-subscriber.ts

```diff
@@ -100,6 +106,11 @@ export class ScrimEventSubscriber {
             this.logger.warn(`Scrim not found for submission ${payload.submissionId}`);
             return;
         }
-        await this.scrimService.resetScrim(scrim.id);
+        await this.scrimService.resetScrim(scrim.id).catch(() => null);
```

Comment:
Caused instances of "scrim was already in x state" without catching. @0c370t look into?

Reply:
Not sure where that specific error would come from; nothing in this function does that, except for the check that the scrim is in ratifying; which _should_ be the case at this point. Not a huge fan of just swallowing the error for the sake of swallowing it.

Reply:
That seem better?